apps/moment.c

```diff
@@ -149,10 +152,10 @@ gkyl_moment_app_new(struct gkyl_moment *mom)
 
   // check if we should update sources
   app->update_sources = 0;
-  if (app->has_field && ns>0) {
+  //if (app->has_field && ns>0) {
```

Comment:
Maybe we get rid of these commented out lines and we just note in the comments that we are always running with sources?